johnwayodi/store-manager
# #161164308 Add Product Details view for Admin

#### What does this PR do?
Adds a Product detail view for the Admin where Admin can update or delete product in context
#### Description of Task to be completed?
Ensure the Update and Delete Buttons achieve their respective funtionality
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, store-manager by default
3 `git checkout ft-all-products-admin-161163057`
4 open product--detail-admin.html on browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161164308
#### Screenshot
![product-details-admin](https://user-images.githubusercontent.com/10683903/46834301-b5ef5080-cdb3-11e8-8423-16981e4af951.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
